### ISSUE TITLE
Provide better examples and documentation for JSON and YAML input

### ISSUE THREAD

**Issue by sobolevnrm** [Electrostatics/apbs#257]
The JSON documentation exists, it just isn't very good.  [This section of the manual](https://apbs.readthedocs.io/en/latest/using/input/new/index.html) provides an overview of the keys and values expected in JSON- or YAML-format input.  Sorry we don't have more direct examples -- I'll create an issue to generate these in the future.

_Originally posted by @sobolevnrm in https://github.com/Electrostatics/apbs/issues/256#issuecomment-1326628329_

**Comment by sobolevnrm** [comment]
These examples should include specific invocation instructions per https://github.com/Electrostatics/apbs/issues/256#issuecomment-1327095557.

**Comment by sobolevnrm** [comment]
No.  However, we welcome community help to improve our documentation.  Thank you.

**Comment by supernovaZhangJiaXing** [comment]
Can apbs program directly read yaml file as PBSA calculation input, instead of old plain text format?

**Comment by sobolevnrm** [comment]
This is high on my to-do list but I am not sure when I'll be able to get started.
